packages/api/endpoints/elasticsearch.js

```diff
@@ -96,6 +96,10 @@ async function reindex(req, res) {
     }
   }
 
+  if (sourceIndex === destIndex) {
```

Comment:
Given there's logic above around L87/75 checking the existence of the the source/destination index, should this be up around L60?

Reply:
Hmm, apologies, the logic that prompted the question was the block after source/dest indexes are set.   That comment shouldn't read L60 -- L85 would be :100: though.

Reply:
0646b1af673124fccdb4ecaffb21d14aee3ddb01  👍🏻

Reply:
Thanks for the update, and the added unit looks good.